lib/startMenu.ex

```diff
@@ -0,0 +1,16 @@
+defmodule StartMenu do
+  def get_user_input do
+    String.to_integer(String.trim(IO.gets("Please enter a number: ")))
+  end
+
+  def display_instructions do
+    IO.puts("Welcome to Elixir Tic-Tac-Toe.")
+    IO.puts("You are the first player.")
+    IO.puts("When prompted by the computer, enter a number to play in a space.")
+    IO.puts("The game will end when there is a win.")
+  end
+
+  def display_win(marker) do
+    IO.puts("Player #{marker} has won!")
+  end
```

Comment:
Two functions with a _display_ prefix in the name outside of the _Display_ module feels like a smell. Maybe combine _Display_ and _StartMenu_ into _UI_ module. Thoughts?

Reply:
I completely agree. I think most of the logic for the display should be located in the display file. Same goes for the game_state file.